modules/@apostrophecms/ui/ui/apos/scss/imports.scss

```diff
@@ -12,4 +12,5 @@
 @import './_type';
 @import './_utilities';
 @import './_zindex';
-@import 'Modules/@apostrophecms/login/scss/login'
+@import 'Modules/@apostrophecms/login/scss/login';
+@import 'Modules/@apostrophecms/notification/scss/notification';
```

Comment:
This shouldn't be necessary. Why aren't the styles in the `AposNotification.vue` component?

Reply:
I see. That files has the styles for the notifications wrapper. Since the notifications component isn't scoped it could still go in AposNotification.vue, but this also makes sense since it's in the related directory. 👍

Reply:
Exactly